calibrate.py

```diff
@@ -22,27 +22,14 @@ def acquire_files():
     return sample_measurements
 
 
-def calibration_check(spectrum):
-    '''
-    calibration_check will search for certain peaks that are expected to occur
-    in every measured spectra. The energies it searches for are based on peaks
-    that occur in background radiation. Once these peaks are found, it
-    compares the energy of that peak to the actual energy the peak should be
-    at. This check is based on expected detector resolution and if the energy
-    deviates to far from the expected value (if beyond half a FWHM), then
-    calibration_check sends a message indicating a fix is needed. Only Spectra
-    are taken as input.
-    '''
-    E0 = spectrum.energy_cal[0]
-    Eslope = spectrum.energy_cal[1]
-    energy_axis = E0 + Eslope*spectrum.channel
-
+def peak_offsets(spectrum):
```

Comment:
Add a docstring describing what this function does and returns.
(Docstring is formatted with triple quotes, 
```
"""
Multi-line description...
"""
```
)